community-operators/apicurio-registry/apicurio-registry-operator.package.yaml

```diff
@@ -1,5 +1,7 @@
 packageName: apicurio-registry
-defaultChannel: alpha
+defaultChannel: 2.x
 channels:
   - name: alpha
     currentCSV: apicurio-registry.v0.0.4-v1.3.2.final
+  - name: 2.x
+    currentCSV: apicurio-registry.v1.0.0-v2.0.0.final
```

Comment:
This line should be `currentCSV: apicurio-registry-operator.v1.0.0-v2.0.0.final`

Reply:
Or change it in cvs to `apicurio-registry.v1.0.0-v2.0.0.final`

Reply:
Right! I'll update it ASAP